biot_savart_integrators/generic/integrand.py

```diff
@@ -13,6 +14,16 @@
 def biot_savart_integrand(r, r_j, j, spatial_dim):
     R = r - r_j
     numerator = cross(j, R, spatial_dim)
-    denominator = (R**2).sum(dim=spatial_dim)**(3/2.)
+    denominator = (R**2).sum(dim=spatial_dim) ** (3 / 2.)
```

Comment:
Are you sure `3 / 2` is more readable? Since it's supposed to be a fraction, I find the original more readable.